features/api_services_list.feature

```diff
@@ -0,0 +1,395 @@
+Feature: Services list api
+
+  @uses.config.contract_token
+  Scenario Outline: Get services list when unattached/attached as non-root
```

Comment:
Given that the test is expecting the api to work exactly the same, it can be simplified into a single scenario, where you add a line before checking the actual text. We have something like
`And I verify root and non-root ``<command>`` calls have the same output`
So you DRY